.github/workflows/github.yml

```diff
@@ -282,6 +295,43 @@ jobs:
         echo "Waiting..."; counter=0; while pgrep XProtect && ((counter < 20)); do sleep 3; ((counter++)); done
         pgrep XProtect || true
 
+    - name: Install Inno Setup 6+
+      if: ${{ startsWith(matrix.platform, 'msvc') }}
+      run: |
+        $defaultPath = "$env:ProgramFiles`(x86`)\Inno Setup 6\ISCC.exe"
+        if (!(Test-Path $defaultPath)) {
+          Write-Host "Inno Setup 6 not found. Downloading and installing..."
+          $installArgs = @("/VERYSILENT", "/NORESTART")
+          Invoke-WebRequest -Uri "https://files.innosetup.nl/innosetup-6.4.3.exe" -OutFile "isetup.exe"
```

Comment:
IS is already installed on the runner: https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md

Reply:
was trivial to find out: https://github.com/kambala-decapitator/workflow-test/actions/runs/16346361412/workflow and https://github.com/kambala-decapitator/workflow-test/actions/runs/16346361412/job/46180999206